### PR TITLE
Prep for `master` to `main` rename

### PR DESCRIPTION
**Pull request by devpow112**
Was gonna need to make branching changes for the Code Artifact work anyways so thought I'd take care of the `master` to `main` shuffle while I was at it (since I've been asked a few times).

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -2,8 +2,10 @@ name: Release
 on:
   push:
     branches:
-      - master
+      - main
+      - main-lit-1.0 # needed for polymer conversion
       - polymer # needed for polymer conversion
+      - polymer-lit-1.0 # needed for polymer conversion
       - release/*
```
Comment (devpow112) [inline .github/workflows/release.yml:9]: Need to maintain the lit 1.0 version of rubrics for a bit longer so set-up branches based on their last releases.

**Comment by devpow112** [comment]
Once this is merged you can run the following to rename your `master` branch to `main` locally.

```console
git branch -m master main
git fetch origin
git branch -u origin/main main
git remote set-head origin -a
```

**Comment by devpow112** [comment]
Gonna wait till next sprint to merge this

File: .github/workflows/update-dependencies.yml [hunk 2]
```diff
@@ -2,9 +2,6 @@ name: Update Dependencies
 on:
   schedule:
     - cron: '00 10 * * 1-5' # Mon-Fri 6:00am EDT/5:00am EST
-  push:
-    branches: master
-    paths: .github/workflows/update-dependencies.yml # when file changed
```
Comment (devpow112) [inline .github/workflows/update-dependencies.yml:7]: Caused more issues then it was worth. This can be manually triggered still.

**Review by mpharoah-d2l — APPROVED** [review]

**Pull request merged**